rai/models.go

```diff
@@ -456,6 +456,20 @@ type SnowflakeDataStream struct {
 }
 
 type createSnowflakeDataStreamRequest struct {
+	Snowflake struct {
+		Database  string `json:"database"` // native app db
```

Comment:
same as other PR, we don't need Database and Schema (previously for DB Link) and Role and Warehouse (previously for executing data stream SQL on the integration services?)

Reply:
I have removed db, schema, role and warehouse.